src/helpers/useAppNavigation.ts

```diff
@@ -46,9 +47,13 @@ export default function useAppNavigation() {
   );
 
   const navigateToCommunity = useCallback(
-    (community: CommunityView) => {
+    (community: CommunityView | Community) => {
       pushRouteIfNeeded(
-        buildGeneralBrowseLink(`/c/${getHandle(community.community)}`),
+        buildGeneralBrowseLink(
+          `/c/${getHandle(
```

Comment:
Can you use `buildCommunityLink` from `appLinkBuilder`? Like `buildGeneralBrowseLink(buildCommunityLink(community))`.

(Sorry I know it's not done everywhere, I'm slowly switching over to it.)

You can make `buildCommunityLink` take a `CommunityView` if it would help!

Reply:
Sure, no problem.